lib/include/jxl/codestream_header.h

```diff
@@ -346,32 +348,47 @@ typedef struct {
 } JxlBlendInfo;
 
 /** The information about layers.
- * If coalescing is enabled (default), this can be ignored.
+ * When decoding, if coalescing is enabled (default), this can be ignored.
+ * When encoding, these settings apply to the pixel data given to the encoder,
+ * the encoder could choose an internal representation that differs.
  */
 typedef struct {
-  /** Horizontal offset of the frame (can be negative, always zero if coalescing
-   * is enabled)
+  /** Whether cropping is applied for this frame. If false, crop_x0, crop_y0
+   * are assumed to be zero, and xsize and ysize are assumed to match the main
+   * image dimensions. When decoding, if coalescing is enabled (default), this
+   * is always false, regardless of the internal encoding in the JPEG XL
+   * codestream.
+   */
+  JXL_BOOL have_crop;
+
+  /** Horizontal offset of the frame (can be negative). Only used if have_crop
```

Comment:
```suggestion
  /** Horizontal offset of the frame (can be negative). Ignored by the encoder if have_crop
    * is false; when decoding with coalescing enabled or if have_crop is false, this is always equal to 0.
```

Reply:
I've instead removed this part, since have_crop already explains this